--- configure	2022-04-15 00:16:11.000000000 +0200
+++ configure.patched	2022-04-22 09:20:15.000000000 +0200
@@ -28920,7 +28920,7 @@
 ## However, although the gcc docs do not mention it, it seems s390/s390x
 ## also supports and needs -fPIC
 ## See <https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1004747> for alpha.
-    alpha*|sparc*|ppc64*|powerpc64*|s390*)
+    aarch64*|alpha*|sparc*|ppc64*|powerpc64*|s390*)
       cpicflags="-fPIC"
       ;;
     *)
@@ -28931,7 +28931,7 @@
 fi
 if test "${ac_cv_fc_compiler_gnu}" = yes; then
   case "${host_cpu}" in
-    alpha*|sparc*|ppc64*|powerpc64*|s390*)
+    aarch64*|alpha*|sparc*|ppc64*|powerpc64*|s390*)
       fpicflags="-fPIC"
       ;;
     *)
@@ -28947,7 +28947,7 @@
 esac
 if test "${GXX}" = yes; then
   case "${host_cpu}" in
-    alpha*|sparc*|ppc64*|powerpc64*|s390*)
+    aarch64*|alpha*|sparc*|ppc64*|powerpc64*|s390*)
       cxxpicflags="-fPIC"
       ;;
     *)
