--- src/scripts/mkinstalldirs.in	2018-09-25 10:46:51.000000000 +0200
+++ src/scripts/mkinstalldirs.in.patched	2022-06-14 10:14:33.000000000 +0200
@@ -15,7 +15,7 @@
 '
 IFS=" ""	$nl"
 errstatus=0
-dirmode=
+dirmode=2775
 
 usage="\
 Usage: mkinstalldirs [-h] [--help] [--version] [-m MODE] DIR ...
@@ -92,8 +92,10 @@
   *)
     if mkdir -m "$dirmode" -p --version . >/dev/null 2>&1 &&
        test ! -d ./--version; then
-      echo "mkdir -m $dirmode -p -- $*"
-      exec mkdir -m "$dirmode" -p -- "$@"
+      echo "mkdir -m $dirmode -p $*"
+      mkdir -m "$dirmode" -p "$@"
+      echo "chown root:staff $*"
+      chown root:staff "$@"
     else
       # Clean up after NextStep and OpenStep mkdir.
       for d in ./-m ./-p ./--version "./$dirmode";
